src/robusta/core/sinks/incidentio/incidentio_sink_params.py

```diff
@@ -12,6 +12,10 @@ class IncidentioSinkParams(SinkBaseParams):
     base_url: Optional[str] = "https://api.incident.io/v2/"
     token: str
     source_config_id: str
+    severity_alert_label_name: Optional[str] = "robusta_incidentio_severity"
```

Comment:
Would it be better to set the default to `severity` which is the default in prometheus alerts?